lib/identity_cors.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class IdentityCors
+  FEDERALIST_REGEX = %r{^https://federalist-[0-9a-f-]+\.app\.cloud\.gov$}
```

Comment:
oh as long as we're here, we should update this (related: https://github.com/18F/identity-devops/pull/5780)

```suggestion
  FEDERALIST_REGEX = %r{^https://federalist-[0-9a-f-]+\.pages\.cloud\.gov$}
```